src/main/java/com/teamcautionrobotics/robot/Cargo.java

```diff
@@ -21,7 +23,10 @@ private CargoMoverSetting(double power) {
     // Attached to mechanum wheels in front of the robot that allow for more driver
     // error.
     private final Solenoid funnelRollerDeployer;
-    private final Solenoid exitFlapDeployer;
+    private Solenoid exitFlapDeployer;
+    private DoubleSolenoid doubleExitFlapDeployer;
```

Comment:
These exitFlapDeployer references can be made final by assigning the unused one to null in the constructor.